### ISSUE TITLE
Dont farm pokestops and dont catch pokemons

### ISSUE THREAD

**Issue by proxyfox123** [finndev/PokeBuddy#701]
keep saying ] [Navigation] Walking from: 40.7656, -73.972 to: 40.7658, -73.9718 with a distance of: 17 meters
[P - 09:44:48] [Navigation] Walking from: 40.76563, -73.97197 to: 40.76583, -73.97177 with a distance of: 17 meter

dont what happening if anyone knows pls tell me

**Comment by Juan44** [comment]
Servers Are Down Buddy

Just Wait :)

**Comment by Thorben24** [comment]
Maybe Softban. Can you login with your phone / bluestacks and check if you are able to get items by a pokestop or are there maybe no pokestops / no pokemons displayed for you?

**Comment by heffman92** [comment]
It's gotta be the servers. I'm not soft banned and the app is working just not this...just gotta wait I guess

**Comment by proxyfox123** [comment]
yeah same here

**Comment by Deuxgod** [comment]
It's an IP ban. Got the same this morning after trying to snipe a Farfetch'd with two bots running. Was getting that message all the time and couldn't see no pokestops or pokemons in either Necrobot or PokeBuddy. Fired up a VPN and tried to run the bots again and it worked fine. After like 1 hour tried again without the VPN and it was working fine again on the same IP

**Comment by Thorben24** [comment]
If you have the problem: "no pokestops / no pokemons displayed for you?" its for 100% an IP ban. Restart your router in this case, so you get a new IP adress.

**Comment by Deuxgod** [comment]
Restarting router will only work in some cases. For example some cable modems keep the same IP after a restart and require the modem to be off a couple of hours before leasing the IP. In case a router reboot doesn't work setting a VPN and connecting through it will allow you to fire up the bot again.

**Comment by Thorben24** [comment]
Oh ok, wasnt informed about this. I just know dynamic ip technology every big provider in germany is using would be used everywhere. 

I use for my 2 bots meanwhile 2 different proxy server so i dont have this problem anymore.

**Comment by heffman92** [comment]
What's the best VPN to use then? I just downloaded totalvpn

**Comment by JewBagel** [comment]
mines is back to working now maybe it was something with the bot

**Comment by Deuxgod** [comment]
It's not something with the bot. The IP ban gets liffted after some time, however it's seems to be applyied if you use more then one instance per IP

**Comment by heffman92** [comment]
Mine started working again too but then it just cut off and kept walking super slow and not registering Pokemon or anything.

**Comment by JewBagel** [comment]
i can promise you i wasnt ip banned when i first started the bot it worked until i used the incubator feature which was 5 mins ago so how can i be ip banned for 5 mins that wouldnt make sense

**Comment by heffman92** [comment]
Yeah it was working fine for me until the incubator feature came up

**Comment by JewBagel** [comment]
it stopped working again it has to be the incubator

**Comment by JewBagel** [comment]
did it start working for anyone else and stop after about 2 mins

**Comment by heffman92** [comment]
Yeah. I just tried the VPN and it didn't even let me log in. It's gotta be the new update. Grrrr

**Comment by Deuxgod** [comment]
Look, I was using Necrobot and PokeBuddy at same time, both with different accounts. When I tried to snipe a pokemon with the account I was using in PokeBuddy I got an IP ban since the account on NecroBot also stopped working. Then I fired up a VPN and the bots started to work normally. If I turn off the VPN the accounts get back to the same state of not finding pokestops or pokemons. It has to be an IP ban that is being applyed by Niantic now. How many bot instances are you running? One or more?

**Comment by JewBagel** [comment]
one im not running any other bots pokebuddy is my go to been working perfect now with this update that has changed .

**Comment by heffman92** [comment]
Yeah I've only been running pokebuddy as well. Worked fine until that incubator message popped up

**Comment by Deuxgod** [comment]
Well, try to disable the Incubator option and run it again in some minutes. It can be something with it that is alerting the servers to issue an IP ban. I'll conduct some more testing once my IP ban is lifted, for now I'll keep using my VPN to bot.

**Comment by JewBagel** [comment]
ive tried the incubator is turned off everytime i restart pokebuddy so im not sure at this point. id gladly use a vpn but have had no luck finding a legit free vpn on pc maybe you can help?

**Comment by heffman92** [comment]
How can u turn the incubator off? And yeah help us with a good VPN cuz I can't find a legit one either

**Comment by Deuxgod** [comment]
http://www.ownedcore.com/forums/pokemon-go/pokemon-go-hacks-cheats/570331-quick-ip-unban-vpn-ip-no-ban.html

Follow this tutorial to set up a proper VPN. To turn the incubator off just check in the settings, it's there.

**Comment by heffman92** [comment]
It worked. Thanks

**Comment by JewBagel** [comment]
still not working for me which server are you using

**Comment by Thorben24** [comment]
Are you guys not in a position to change your ip adress by restarting or relogging your router/modem ?

**Comment by Thorben24** [comment]
@JewBagel 

Whats your exact problem? Are you maybe softbanned?

**Comment by JewBagel** [comment]
nope not softbanned still not seeing any pokestops on pokebuddy

**Comment by JewBagel** [comment]
ISP:	Open Computer Network
City:	Iizuka
Region:	Fukuoka
Country:	Japan

even with a vpn active which thank you very much for no pokestops no pokemon nothing

**Comment by Deuxgod** [comment]
Try another VPN with the bot off

**Comment by heffman92** [comment]
I used the VPN followed the directions. It worked for a little bit but now just took a crap again

**Comment by Deuxgod** [comment]
Maybe someone logged on the same VPN to bot also. Try another VPN

**Comment by JewBagel** [comment]
maybe it happened to you also where for a few mins it was working just like for me 

Deuxgod do you mean try another bot with the vpn off lol

**Comment by JewBagel** [comment]
nah its 100 percent this bot i just tried another bot and it works just fine finds pokestops and pokemon

**Comment by Phillip0810** [comment]
Sorry I just made an account to say everyone in this chat which is talking about it being an ip ban is talking crap. UNLESS you receive 0xp from a pokestop then it is an ip softban.

HOWEVER the duplication of distance travelled or duplication of anything text based within PokeBuddy is due to the PokeBuddy bot.

To fix duplicated text. Shut the bot and reopen it. If you are jumping a distance when doing this as the bot likes to slowly travel away from your original destination then you will get soft banned.

However if you are botting in one area and stop/start the bot it will go back to its original location you put it and might not find pokemon in that area because you've already farmed they're and Pokemon may not have respawned.

Problem delt with

**Comment by Phillip0810** [comment]
Just ran the bot and i want to add that due to the UI no longer updating the pokestops do not reload as you continue to farm. Due to this the bot then spams 17 meters of travel. Once you get too far away from where you started you then press Stop. 

Stop does nothing.

Close the bot and reopen it and start it up again but this time because you've teleported back by closing and opening the bot you become soft banned and it then continues to occur.

The reason why this is happening is due to the developer updating the bot so the UI isn't updating which is why were aren't able to let the bot travel
